scripts/build/package.js

```diff
@@ -76,11 +80,28 @@ data.icons.forEach(icon => {
   icon.slug = filename;
   icons.push(icon);
 
+  const iconObject = iconToObject(icon)
```

Comment:
Minor, but for consistency this should have a semicolon (we should probably just add prettier to this repo, but that's for a separate PR)

```suggestion
  const iconObject = iconToObject(icon);
```